### PR TITLE
Key/value set indexes benchmark [ECR-735]

### PR DESCRIPTION
**Pull request by DarkEld3r**

### CODE REVIEW

File: exonum/benches/criterion/indexes.rs [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+            .throughput(|s| Throughput::Bytes(*s as u32))
+            .plot_config(PlotConfiguration::default().summary_scale(AxisScale::Logarithmic)),
+    );
+    fs::remove_dir_all(tempdir.path()).unwrap();
+    let path = tempdir.path().to_path_buf();
+    c.bench(
+        "contains",
+        ParameterizedBenchmark::new("value_set_index: contains_by_hash", move |bencher, key_size| {
+            value_set_contains_by_hash(path.clone(), bencher, key_size)
+        }, (6..16).map(|i| pow(2, i)))
```
Comment (vldm) [inline exonum/benches/criterion/indexes.rs]: we can move this iterator somewhere, it's looking like reusable

Comment (vldm) [reply exonum/benches/criterion/indexes.rs]: Probably benchmark initialisation can be mooved too

```rust
fn create_benchmark<F>(group:&str, name:&str, F)
where F:FnMut(&Path, &mut Bencher, &usize) 
```

Or you didn't found it usefull?

**Comment by DarkEld3r** [comment]
@vldm It is "WIP" for a reason. :upside_down_face:

**Comment by DarkEld3r** [comment]
Outdated.

**Pull request closed without merging**